sdk/entra/WebJobs.Extensions.AuthenticationEvents/test/WebJobs.Extensions.AuthenticationEvents.Tests.csproj

```diff
@@ -0,0 +1,72 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+	<PropertyGroup>
+		<TargetFramework>netcoreapp3.1</TargetFramework>
+		<IsTestProject>true</IsTestProject>
+		<SignAssembly>True</SignAssembly>
+		<AssemblyOriginatorKeyFile>FriendAssemblies.snk</AssemblyOriginatorKeyFile>
+	</PropertyGroup>
+
+	<PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+		<DocumentationFile>WebJobs.Extensions.CustomAuthenticationExtension.Tests.xml</DocumentationFile>
+	</PropertyGroup>
+
+	<ItemGroup>
+		<Compile Remove="Schema\OnTokenIssuanceStart\stable\**" />
+		<EmbeddedResource Remove="Schema\OnTokenIssuanceStart\stable\**" />
+		<None Remove="Schema\OnTokenIssuanceStart\stable\**" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<None Remove="Payloads\TokenIssuanceStart\CloudEventActionResponse.json" />
+		<None Remove="Payloads\TokenIssuanceStart\ConversionPayload.json" />
+		<None Remove="Payloads\TokenIssuanceStart\InvalidActionResponse.json" />
+		<None Remove="Payloads\TokenIssuanceStart\NoActionResponse.json" />
+		<None Remove="Payloads\TokenIssuanceStart\Preview10012021\ActionResponse.json" />
+		<None Remove="Payloads\TokenIssuanceStart\Preview10012021\Cloud.json" />
+		<None Remove="Payloads\TokenIssuanceStart\Preview10012021\ExpectedCloudPayload.json" />
+		<None Remove="Payloads\TokenIssuanceStart\Preview10012021\ExpectedPayload.json" />
+		<None Remove="Payloads\TokenIssuanceStart\Preview10012021\RequestSchema.json" />
+		<None Remove="Payloads\TokenIssuanceStart\Preview10012021\ResponseSchema.json" />
+		<None Remove="Payloads\TokenIssuanceStart\QueryParameters.json" />
+	</ItemGroup>
+
+	<ItemGroup>
+		<EmbeddedResource Include="Payloads\OpenApi\OpenApiDocument.json" />
+		<EmbeddedResource Include="Payloads\OpenApi\OpenApiDocumentMerge.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\CloudEventActionResponse.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\ConversionPayload.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\InvalidActionResponse.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\NoActionResponse.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\Preview10012021\ActionResponse.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\Preview10012021\Cloud.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\ExpectedPayload.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\Preview10012021\ExpectedPayload.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\Preview10012021\RequestSchema.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\Preview10012021\ResponseSchema.json" />
+		<EmbeddedResource Include="Payloads\TokenIssuanceStart\QueryParameters.json" />
+	</ItemGroup>
+
+	<ItemGroup>
```

Comment:
These will fail CI;  they'll need to use the central versions.

Reply:
Thanks @jsquire These passed CI (I think), This contain or test payloads requests and responses. I'm a little unclear on what I need to do here.

Reply:
This passed the Azure.Core CI checks, which aren't running against your code but were triggered by the central package version updates.

Reply:
You'll need to do the following to bootstrap a pipeline for this library:
https://dev.azure.com/azure-sdk/internal/_wiki/wikis/internal.wiki/117/Setting-up-a-new-Unified-Pipeline

Reply:
Done, Not sure what you 'central versions' are. They are embedded resources.